Src/Support/GoogleApis.Auth.DotNet4/OAuth2/AuthorizationDialog.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Google.Apis.Auth.OAuth2.Responses;
```

Comment:
Space between System and Google usings.
In general our using statements look like:

using System
[empty line]
using 3rd party libraries, e.g. Json.
[empty line]
using Google....

All sections are sorted alphabetically.